Content.Shared/Construction/Conditions/TileType.cs

```diff
@@ -28,7 +28,7 @@ public bool Condition(EntityUid user, EntityCoordinates location, Direction dire
             var tile = tileFound.Value.Tile.GetContentTileDefinition();
             foreach (var targetTile in TargetTiles)
             {
-                if (tile.ID == targetTile)
+                if (tile.ID.StartsWith(targetTile))
```

Comment:
This looks pretty smelly, better off being a flag or the likes.

Reply:
I don't know what we need here really, not enough use cases. This class is only used for catwalk, grille and solar assembly.